editor/editor_export.cpp

```diff
@@ -1760,8 +1760,8 @@ void EditorExportPlatformPC::get_preset_features(const Ref<EditorExportPreset> &
 }
 
 void EditorExportPlatformPC::get_export_options(List<ExportOption> *r_options) {
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE), ""));
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/release", PROPERTY_HINT_GLOBAL_FILE), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.exe,*.apk,*.zip"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/release", PROPERTY_HINT_GLOBAL_FILE, "*.exe,*.apk,*.zip"), ""));
```

Comment:
This class is `EditorExportPlatformPC` and is only used for Windows and Linux, so `.apk` and `.zip` are not relevant. The other platforms have their own hints:
```
$ rg --sort path 'ExportOption.*"custom_template/debug"'
editor/editor_export.cpp
1763:   r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE), ""));

platform/android/export/export_plugin.cpp
1680:   r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.apk"), ""));

platform/iphone/export/export_plugin.cpp
76:     r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.zip"), ""));

platform/javascript/export/export_plugin.cpp
327:    r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.zip"), ""));

platform/osx/export/export_plugin.cpp
72:     r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.zip"), ""));

platform/uwp/export/export_plugin.cpp
69:     r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, "*.zip"), ""));
```

So only `.exe` seems relevant here for Windows, but since it will make the experience to select Linux templates more cumbersome, I'm not sure the change is good to make as is.

@bruvzg and @aaronfranke have refactored some of this code to make Windows and Linux-specific parts properly defined in their platform export plugin, maybe this could be too.

Reply:
Yeah, this logic should be refactored to be separate in Windows and Linux export plugins, and this PR isn't good as-is.

Reply:
Though without major refactoring, maybe this would work just fine (untested):
```suggestion
	String ext_filter = (get_os_name() == "Windows") ? "*.exe" : "";
	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE, ext_filter), ""));
	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/release", PROPERTY_HINT_GLOBAL_FILE, ext_filter), ""));
```

Reply:
For the record, I suggested to @taigi100 on chat to use my proposed ad hoc solution for now as it would be suitable for `3.x`, while a future refactor wouldn't be backported.